packages/svelte-vscode/tsconfig.json

```diff
@@ -6,6 +6,7 @@
         "declaration": true,
         "outDir": "dist",
         "sourceMap": true,
-        "composite": true
+        "composite": true,
+        "target": "ES5"
```

Comment:
I was getting warning/error on https://github.com/sveltejs/language-tools/blob/86477382b4f5bb76effd9736147652e8a0513463/packages/svelte-vscode/src/CompiledCodeContentProvider.ts#L53 so I addes target, that fix it